News 2021 11 (#298)
ropensci/roweb3: content/blog/2021-11-30-ropensci-news-digest-november-2021/index.md

---
slug: "ropensci-news-digest-november-2021"
title: 'rOpenSci News Digest, November 2021'
author:
  - The rOpenSci Team
date: '2021-11-30'
tags:
  - newsletter
description: keywords from the content
output:
  html_document:
    keep_md: yes
params:
  last_newsletter: "2021-10-22"
---

<!-- Before sending DELETE THE INDEX_CACHE and re-knit! -->

Dear rOpenSci friends, it's time for our monthly news roundup!
<!-- blabla -->
You can read this post [on our blog](/blog/2021/11/30/ropensci-news-digest-november-2021).
Now let's dive into the activity at and around rOpenSci!

## rOpenSci HQ

> ### Community call. @ropensci-review-bot help!

Reviewer: I was a bit confused by this heading, perhaps a colon will connect the two a bit better?
```suggestion
### Community call: @ropensci-review-bot help!
```

Speakers [Arfon Smith](https://www.arfon.org/), [Karthik Ram](/author/karthik-ram), [Maëlle Salmon](/author/maëlle-salmon) and [Noam Ross](/author/noam-ross) will discuss how rOpenSci has worked with [The Journal of Open Source Software (JOSS)](https://joss.theoj.org/) to extend their approach of [chatops-driven publishing](https://www.arfon.org/chatops-driven-publishing) into a new GitHub chat-bot that manages our editorial process: assigning tasks, tagging issues, running tests on software submissions, and returning reports to reviewers and editors, all from the comfort of a GitHub issue comment. Chat-ops automation has been critical to JOSS’s success, and shows huge promise for automating even more tedious tasks like frequent software checks and compliance with best practices. We will show how our new review bot improves the author and editor experience, **demonstrate how other teams can use and customize the bot**, and discuss how this approach can be extended to tackle other automation, code review, and reproducibility challenges.

> Lots of time for Q & A during the session.

Reviewer: During, or at the end?
```suggestion
Come join the conversation, we'll have lots of time for Q & A during the session.
```

Reviewer 2: I boorishly stepped in and edited Steffi's suggested change.

### rOpenSci help wanted issues

> Help Wanted issues have a home: our website's new [help-wanted page](/help-wanted/).

Reviewer: I'm excited :wink:
```suggestion
Help Wanted issues have a home: our website's new [help-wanted page](/help-wanted/)!
```

> This page lists ALL recent(ish) issues from rOpenSci R packages labelled "help wanted". We'll still [tweet new issues](https://twitter.com/search?q=from%3Aropensci%20%5Bhelp%20wanted%5D&f=live)!

Reviewer: ```suggestion
This page lists ALL recent(ish) issues from rOpenSci R packages labelled "help wanted". We'll still [tweet new issues](https://twitter.com/search?q=from%3Aropensci%20%5Bhelp%20wanted%5D&f=live), but this is a great place to see what's out there that needs your help.
```

> 
> - Sort e.g, good first issue;
> - Search e.g., specific packages.

Reviewer: In the tweet I was pretty brief due to character constraints, but I think we can expand a bit more here. 
```suggestion

On the page you can easily find issues relevant to you by:  

- Sorting (e.g, find all the issues labelled "good first issue");
- Searching (e.g., look for issues in your favourite packages)
```

### Next co-working sessions

Join us for social coworking & office hours monthly on 1st Tuesdays! Hosted by Steffie LaZerte, Nick Tierney, and Stefanie Butland. Everyone welcome. No RSVP needed. Consult our [Events](/events) page to find your local time and how to join.

Our next sessions are

> * [Monday, 06 December 2021 17:00 PST](/events/coworking-2021-12/);
> * [Tuesday, 04 January 2022 09:00 PST](/events/coworking-2022-01/).

Reviewer: Feel free to omit if you think this is too redundant!
```suggestion
* [Monday, 06 December 2021 17:00 PST](/events/coworking-2021-12/) (9 AM Australian Western);
* [Tuesday, 04 January 2022 09:00 PST](/events/coworking-2022-01/) (9 AM North American Pacific).
```

### Other topics

* [How to Cite R and R Packages](/blog/2021/11/16/how-to-cite-r-and-r-packages) by Steffi LaZerte. A short primer on how to cite R and R packages to support developers and ensure repeatable science.

## Use cases

<!--` snakecase::to_sentence_case(english(length(usecases)))` use cases of our packages and resources have been reported since we sent the last newsletter.-->

Explore [other use cases](/usecases) and [report your own](https://discuss.ropensci.org/c/usecases/10)!

## Call for maintainers

<!--IF CALL
* [our guidance on _Changing package maintainers_](https://devguide.ropensci.org/changing-maintainers.html)
* [our _Package Curation Policy_](https://devguide.ropensci.org/curationpolicy.html)

IF NO CALL
There's no open call for new maintainers at this point but you can refer to our [contributing guide](https://contributing.ropensci.org/) for finding ways to get involved!

> As the maintainer of an rOpenSci package, feel free to contact us on Slack or email `info@ropensci.org` to get your call for maintainer featured in the next newsletter. -->

Reviewer: I'm sure you're on top of it, but this comment is a reminder to update this section :)

> ## Package development corner

Reviewer: This section always amazing me, thanks so much!
